playbooks/roles/edxapp/files/update_theme.sh

```diff
@@ -17,7 +17,7 @@ if [ "$1" == "true" ] || [ "$1" == "True" ]; then
 	sudo git clone https://github.com/microsoft/edx-theme.git themes -b oxa/master.fic
```

Comment:
the theming repository details are available via configuration. We may not want to hard code it here.